docs/en/observability/profiling-probabilistic-profiling.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[profiling-probabilistic-profiling]]
+= Configure probabilistic profiling
+
+beta::[]
+
+Configuring probabilistic profiling is a viable option to reduce storage costs.
+
+The option `-probabilistic-threshold` expects an unsigned integer as argument. If set to a value
+between 1 and 99 will enable probabilistic profiling. Every probabilistic interval a random number
+between 0 and 99 is chosen. If the given probabilistic threshold is greater than this random
+number, the agent will collect profiles from this system for the duration of the interval.
+
+The option `-probabilistic-interval` expects a time duration and defines the time interval for
+which probabilistic profiling will be enabled or disabled. Its default value is 1 minute.
+
+The following example shows how to configure the Universal Profiling Agent with a threshold of 50
+and an interval of 2 minutes and 30 seconds:
+
+[source,bash]
+----
+sudo pf-host-agent/pf-host-agent -probabilistic-threshold=50 -probabilistic-interval=2m30s ...'
+----
```

Comment:
I would add a real-world example here at the end, describing what this configuration amounts to in practical terms. For instance, something like:

"Over time and across the agent fleet, this configuration is expected to result in  ~50% of deployed agents collecting profiles at any given time. This reduces storage costs by accepting the trade-off of not having profile collection enabled all the time on all agents."

Reply:
added suggestion.